### ISSUE TITLE
May I ask where the model parameters were printed and which specific code segment

### ISSUE THREAD

**Issue by moshengmao** [jsvir/vad#5]
When I run the train.py，I can get  the information below，but I want to know which code generates them：

>   | Name              | Type                             | Params
-----------------------------------------------------------------------
0 | spec_augmentation | SpectrogramAugmentation          | 0     
1 | crop_or_pad       | CropOrPadSpectrogramAugmentation | 0     
2 | preprocessor      | AudioToMFCCPreprocessor          | 0     
3 | fs                | ConvASREncoder                   | 7.8 K 
4 | encoder           | ConvASREncoder                   | 68.0 K
5 | decoder           | ConvASRDecoderClassification     | 387   
6 | loss              | CrossEntropyLoss                 | 0     
7 | _accuracy         | TopKClassificationAccuracy       | 0     
-----------------------------------------------------------------------
76.2 K    Trainable params
0         Non-trainable params
76.2 K    Total params
0.305     Total estimated model params size (MB)

Maybe `trainer= pl.Trainer(**cfg.trainer)`？？

**Comment by jsvir** [comment]
Hi,
sorry for the late response, I think the trainer of pytorch lightning prints it.
